From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Stained Glass OS <ke7oxh@gmail.com>
Date: Wed, 14 Oct 2026 20:54:29 -0700
Subject: [PATCH] infosoft: class factory LockServer succeeds (3269)

LockServer returned E_NOTIMPL, which the documentation does not list; it now returns S_OK.

diff --git a/dlls/infosoft/infosoft_main.c b/dlls/infosoft/infosoft_main.c
index 6a109fa..2efc804 100644
--- a/dlls/infosoft/infosoft_main.c
+++ b/dlls/infosoft/infosoft_main.c
@@ -96,8 +96,11 @@ static HRESULT WINAPI infosoftcf_fnCreateInstance( LPCLASSFACTORY iface,
 
 static HRESULT WINAPI infosoftcf_fnLockServer(LPCLASSFACTORY iface, BOOL fLock)
 {
-    FIXME("%p %d\n", iface, fLock);
+    TRACE("%p %d\n", iface, fLock);
+#ifdef SG_MUTANT_INFOSOFT
     return E_NOTIMPL;
+#endif
+    return S_OK;
 }
 
 static const IClassFactoryVtbl infosoft_cfvt =
